frontend/tailwind.config.ts

```diff
@@ -1,8 +1,8 @@
-const defaultTheme = require("tailwindcss/defaultTheme")
-const plugin = require("tailwindcss/plugin")
+import defaultTheme from "tailwindcss/defaultTheme";
+import plugin from "tailwindcss/plugin";
 
-const { SCREENS } = require("./src/constants/screens")
-const { Z_INDICES } = require("./src/constants/z-indices")
+import { SCREENS } from "./src/constants/screens";
+import { Z_INDICES } from "./src/constants/z-indices";
```

Comment:
Openverse uses ESLint which makes sure that formatting is always consistent. We don't use `;` at the end of lines.